trio/_core/_generated_run.py

```diff
@@ -103,7 +109,9 @@ def reschedule(task, next_send=_NO_SEND):
         raise RuntimeError("must be called from async context")
 
 
-def spawn_system_task(async_fn, *args, name=None, context=None):
+def spawn_system_task(async_fn: Callable[..., Awaitable[object]], *args:
+    Any, name: object=None, context: (contextvars.Context | None)=None
+    ) ->trio.lowlevel.Task:
```

Comment:
Fix formatting

Reply:
This is auto-generated, it's more a problem with the `astor` package.